cdm/core/src/main/java/ucar/nc2/filter/Checksum32.java

```diff
@@ -19,7 +19,16 @@
 public class Checksum32 extends Filter {
 
   public enum CType {
-    FLETCHER, ADLER, CRC
+    FLETCHER("fletcher32", 3), ADLER("adler32", -1), CRC("crc32", -1);
```

Comment:
Should the second and third types have distinct ids?

Reply:
Fletcher is the only one that has an ID in HDF5, so the others should only be using a name